gensim/models/atmodel.py

```diff
@@ -354,7 +369,21 @@ def extend_corpus(self, corpus):
             self.corpus.extend(corpus)
 
     def compute_phinorm(self, expElogthetad, expElogbetad):
-        """Efficiently computes the normalizing factor in phi."""
+        """Efficiently computes the normalizing factor in phi.
+
+        Parameters
+        ----------
+        expElogthetad: numpy.ndarray
+            Value of variational distribution q(theta|gamma).
```

Comment:
Better use latex for this stuff (this will be rendered in documentation)

Reply:
What would be the representation for this in LaTex?

Reply:
@souravsingh look into original paper + [:math: in sphinx](http://www.sphinx-doc.org/en/stable/ext/math.html)

Reply:
Would it be like this- `q(\theta|\gamma)`

Reply:
@souravsingh maybe, I'm not sure, check original paper.